src/plugins/dashboard/public/dashboard_container/embeddable/api/add_panel_from_library.ts

```diff
@@ -14,10 +14,6 @@ export function addFromLibrary(this: DashboardContainer) {
   this.openOverlay(
     openAddPanelFlyout({
       container: this,
-      onAddPanel: (id: string) => {
-        this.setScrollToPanelId(id);
-        this.setHighlightPanelId(id);
-      },
```

Comment:
Looks like this was dropped, so we no longer have the nice highlight + scroll to behaviour when adding a panel. Was this intentional? 

**Before:**

https://github.com/elastic/kibana/assets/8698078/da3e73c5-9e36-4263-9b4c-324be58d5e1b

**After:**

https://github.com/elastic/kibana/assets/8698078/4aa12d69-83dc-40ff-8f27-c7a3c233c667

Reply:
Consequences of my own actions in d61f44558e9a503105f9d571f12279d9344819fb. 🤦 

Thanks for catching this. I will work on a solution.

Reply:
Fixed in https://github.com/elastic/kibana/pull/179289/commits/a07d6a7cff52cb74245589dcdf029674871a54aa. 

Supporting panel placement strategies in legacy embeddable factories requires using the `IContainer.addNewEmbeddable` method directly rather than `PresentationContainer.addNewPanel` method. Starting with Links, embeddables will register their panel placement strategies with a [registry](https://github.com/elastic/kibana/pull/178670/files#diff-224ab4033e8ec2ac765d4273780d7cba47eb680ea4f753b8201f3d29fdf9274a) that will allow `PresentationContainer` types to place the panel.